src/router/index.js

```diff
@@ -11,6 +14,21 @@ const routes = [
     name: 'home',
     component: Home,
   },
+  {
+    path: '/download',
+    name: 'Download',
+    component: Download,
+  },
+  {
+    path: '/favourites',
+    name: 'Favourites',
+    component: Favourites,
+  },
+  {
+    path: '/layers',
+    name: 'Layers',
+    component: Layers,
+  },
```

Comment:
Can `layers` be the default view? I don't think we need a difference between `home` and `layers` - if someone opens the app we start with the `layers` view